test/e2e/run_signal_test.go

```diff
@@ -112,7 +108,8 @@ var _ = Describe("Podman run with --sig-proxy", func() {
 		Expect(killSession).Should(ExitCleanly())
 
 		session.WaitWithDefaultTimeout()
-		Expect(session).To(ExitWithError(2, "SIGFPE: floating-point exception"))
+		errorMsg := "SIGFPE: floating-point exception"
+		Expect(session).To(Or(ExitWithError(2, errorMsg), ExitWithError(134, errorMsg)))
```

Comment:
@edsantiago Not sure if you like this like that? It is certainly more duplication that I wanted but the alternative is to switch ExitWithError() to accept a list of exit codes which seems much more complicated.

Reply:
Done, the alternative is to patch ExitWithError() to accept `any` as first arg and then use it to pass an int like we do now or an array of int's to allow several exit codes. But this is much more complicated and for a single test overkill.